readthedocs/doc_builder/config.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-"""An API to load config from a readthedocs.yml file."""
+"""An API to load config from a 'readthedocs.yml', 'readthedocs.yaml', or '.readthedocs.yml' file."""
```

Comment:
I think this can remain as `readthedocs.yml file`, is kind of clear enough that the name can be `.rea...` etc. Just a thought.